integrators/ordinary_differential_equations_body.hpp

```diff
@@ -1,16 +1,33 @@
 ﻿
 #pragma once
 
+#include "integrators/ordinary_differential_equations.hpp"
+
+#include <vector>
+
 #include "base/macros.hpp"
 #include "integrators/embedded_explicit_runge_kutta_nyström_integrator.hpp"
-#include "integrators/ordinary_differential_equations.hpp"
 #include "integrators/symmetric_linear_multistep_integrator.hpp"
 #include "integrators/symplectic_runge_kutta_nyström_integrator.hpp"
 
 namespace principia {
 namespace integrators {
 namespace internal_ordinary_differential_equations {
 
+template<typename Position_>
+SpecialSecondOrderDifferentialEquation<Position_>::SystemState::SystemState(
+    std::vector<typename SpecialSecondOrderDifferentialEquation<
```

Comment:
I would expect `std::vector<Position>` to work, as it seems that `Position` would be visible here.  Why is this trick needed?